deformdemo/__init__.py

```diff
@@ -3,61 +3,62 @@
 """ A Pyramid app that demonstrates various Deform widgets and
 capabilities and which provides a functional test suite  """
 
+# Standard Library
+import csv
 import decimal
 import inspect
+import logging
+import pprint
 import random
-import re
```

Comment:
Why were `re` and `extPat` removed?

Reply:
@stevepiercy re is not used here, so it is safe to remove.
extPat was moved few lines down